app/scripts/lib/notification-manager.js

```diff
@@ -3,6 +3,18 @@ import ExtensionPlatform from '../platforms/extension'
 const NOTIFICATION_HEIGHT = 620
 const NOTIFICATION_WIDTH = 360
 
+function getLeftTop() {
+  var a = typeof window.screenX != 'undefined' ? window.screenX : window.screenLeft;
```

Comment:
This function was grabbed from stackoverflow and was a dud, I'll remove it and its not a part of the solution.